h2o-py/tests/testdir_jira/pyunit_pubdev_4995.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import h2o
+from tests import pyunit_utils
+
+from h2o.estimators.aggregator import H2OAggregatorEstimator
+from h2o.exceptions import H2OResponseError
+
+
+DATASET = pyunit_utils.locate("smalldata/extdata/australia.csv")
+
+
+def test_4995():
+    data = h2o.import_file(DATASET)
+
+    raised = False
+    agg = H2OAggregatorEstimator(model_id="aggregator")
+    try:
+        agg.train(training_frame=data, x="minairtemp", y="maxairtemp")
```

Comment:
We should also test that the API provides correct results. Without that we cannot close PUBDEV-4995 (the jira is really about aggregator not being supported in Python).